.storybook/webpack.config.ts

```diff
@@ -0,0 +1,16 @@
+import { Configuration } from 'webpack';
+import { clientConfig } from '../webpack.config';
+
+export default async ({ config } : { config: Configuration }) : Promise<Configuration> => {
+	return {
```

Comment:
The closest type I found was `ConfigurationFactory` in [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/77357b2c3de87fdaee7aa75487b5e767fa26dbec/types/webpack/index.d.ts#L154) but it's not it as the config there is a string.